Components/alert.css

```diff
@@ -0,0 +1,25 @@
+.alert{
+    margin: 1rem 0rem;
+    padding: 1rem 1rem;
+    border-radius: 4px;
+}
+.alert-primary{
```

Comment:
Give at least one line space b/w two declarations.

Reply:
Okay thanks for this suggestion will give spaces in between.